keyboards/paprikman/albacore/readme.md

```diff
@@ -0,0 +1,15 @@
+# Albacore
+
+*A small 7-key macropad featuring hotswap and per-key RGB*
+
+![Albacore](https://github.com/paprikman/my_files/blob/master/albacore_macropad.jpg)
+
+* Keyboard Maintainer: paprikman
+* Hardware Supported: Albacore hotswap pcb
+* Hardware Availability: Private group buy
+
+Build and flash firmware (via support is considered by default):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```